Having hard coded "lib" string in lib_python path, searches for modules 
under /usr/lib/python3.X directory. This would result in an error on 
multilib environment, where base_libdir is other than lib.

Example: On qemux86-64 with base_libdir and libdir as /lib64 and /usr/lib64
respectively, would result in below error when python3 is executed:
root@qemux86-64:~# python3
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: Py_Initialize: Unable to get the locale encoding
ImportError: No module named 'encodings'

Current thread 0x00007f65dbb53700 (most recent call first):
Aborted
root@qemux86-64:~# 

diff -Naurp Python-3.5.2_org/Modules/getpath.c Python-3.5.2/Modules/getpath.c
--- Python-3.5.2_org/Modules/getpath.c	2017-02-17 00:58:28.312583182 +0530
+++ Python-3.5.2/Modules/getpath.c	2017-02-17 01:01:58.984805492 +0530
@@ -502,7 +502,7 @@ calculate_path(void)
     _pythonpath = Py_DecodeLocale(PYTHONPATH, NULL);
     _prefix = Py_DecodeLocale(PREFIX, NULL);
     _exec_prefix = Py_DecodeLocale(EXEC_PREFIX, NULL);
-    lib_python = Py_DecodeLocale("lib/python" VERSION, NULL);
+    lib_python = Py_DecodeLocale(LIB "/python" VERSION, NULL);
 
     if (!_pythonpath || !_prefix || !_exec_prefix || !lib_python) {
         Py_FatalError(
